images/php/fpm/Dockerfile

```diff
@@ -36,9 +37,18 @@ COPY ssmtp.conf /etc/ssmtp/ssmtp.conf
 
 ENV NEWRELIC_VERSION=8.6.0.238
 
-RUN apk del --no-cache curl libcurl \
-    && apk add --no-cache "curl=7.61.1-r2" "libcurl=7.61.1-r2" --repository http://dl-cdn.alpinelinux.org/alpine/v3.8/main/ \
-    && apk add --no-cache fcgi \
+RUN apk del --no-cache curl libcurl
+
+# PHP 5.6 and 7.0 are officially EOL as of January, 2019.
+# This command below pins the last known good curl packages for each version
+RUN if [ ${PHP_VERSION%.*} == "7.3" ] ; then \
+        apk add --no-cache "curl=7.65.1-r0" "libcurl=7.65.1-r0" --repository http://dl-cdn.alpinelinux.org/alpine/v3.10/main/ ; \
```

Comment:
do we need to pin a specific version here? couldn't we just install the newest version? I would like to pin as few packages as possible.

Reply:
Per our conversation in Zoom, I'm going to remove our curl pinning entirely.